SOURCES.md

```diff
@@ -0,0 +1,84 @@
+# Sources
+
+In SmartEvents, a `Source` is a component of a `Processor` that connects to an external source of events (hence the name)
+and, if they are matched by a [Filter](FILTERS.md), automatically injects them in the system.
+
+Processors containing sources are called **"source processors"**.
+
+It is **not possible** to create a processor containing an [Action](ACTIONS.md) and a `Source` at the same time.
+
+When a new source `Processor` is requested using the endpoint `/api/v1/bridges/{id}/processors` you must specify the `Source` to connect to and, optionally, one or more [Filters](FILTERS.md).
+If no filter is specified, all the events coming from the `Source` are injected in the system.
+
+[Transformations](TRANSFORMATIONS.md) are currently not supported in source processors.
+
+## Parameters of a Source
+
+Each `Source` has 2 parameters to specify:
+- `type`: the type of the `Source`. This must be one of the supported `Source` types listed below
+  - Attempting to use an unknown `Source` type will result in an Error from the Bridge API.
+- `parameters`: A key/value map of configuration parameters for the `Source`
+  - Only string for the `key` and `value` of the parameters are supported.
+  - The required parameters are `Source` specific and documented in the list of supported `Sources`
+
+## Supported Source Types
```

Comment:
@kelvah FYI.. this seems to be the only "Source Processor" we need to worry about in the UI at the moment.